data.json

```diff
@@ -0,0 +1,192 @@
+[
```

Comment:
I'm not completely sure if we need this file...

Reply:
I'm not either